README.md

```diff
@@ -1,24 +1,27 @@
 # Homebrew
-Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
+**Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).**
 
 ## What Packages Are Available?
-1. Type `brew search` for a list.
-2. Or visit [braumeister.org](http://braumeister.org) to browse packages online.
-3. Or use `brew search --desc <keyword>` to browse packages from the command line.
+1. Use `brew search` on the command line for a list of packages.
+2. Use `brew search --desc <keyword>` to browse packages from the command line.
+3. Alternatively, you can visit [braumeister.org](http://braumeister.org) to browse packages online.
```

Comment:
How about dropping “Alternatively, you can” and directly starting with “Visit”? Feels also more consistent with the other two items.